### PR TITLE
STL 570 - Update Seth Docs

### PR DESCRIPTION
**Pull request by aludvik**

### CODE REVIEW

File: docs/source/seth_developers_guide/cli_reference.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+-----------------
+.. literalinclude:: /cli/output/seth_account_list_usage.out
+   :language: console
+   :linenos:
+
+seth account create
+-------------------
+.. literalinclude:: /cli/output/seth_account_create_usage.out
+   :language: console
+   :linenos:
```
Comment (chenette) [inline docs/source/seth_developers_guide/cli_reference.rst]: You probably didn't touch "seth account create" in this PR, but I wanted to make a note that the defaults for --nonce and --wait might be wrong (you mentioned this in our conversation last wee).

File: docs/source/seth_developers_guide/permissions.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+see a line print similar to::
+
+    Perms  : -root,+send,+call,+contract,+account
+
+This is because the contract inherited all permissions from the creating
+account, except for root.
+
+Permissions of individual accounts may be modified by any account that has the
+"root" permission. As such, it is important to limit the number of accounts
+that have "root". You can the permissions of other accounts with ``seth
```
Comment (chenette) [inline docs/source/seth_developers_guide/permissions.rst]: Sorry - noticed one more typo.  Change to "You can control the permissions..."

File: docs/source/seth_developers_guide/permissions.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+account
+  Create new externally owned accounts; This only has an effect at the global
+  permissions address and disabling it prevents new external accounts from
+  being created
+
+Each permission can be set to either allow or deny the action for individual
+accounts. If a permission is not set, the permission defaults to the permission
+set at the global permissions address. If the permission is not set at the
+global permissions address, the permission is allowed.
+
```
[the remark this answers is not in the document]
Comment (aludvik) [reply docs/source/seth_developers_guide/permissions.rst]: Update CLI usage

File: docs/source/seth_developers_guide/cli_reference.rst (hunk 1 above)
Comment (aludvik) [reply docs/source/seth_developers_guide/cli_reference.rst]: Updated

File: docs/source/seth_developers_guide/permissions.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  Transfer value from an owned account to another account; Not currently used
+
+call
+  Execute a deployed contract
+
+contract
+  Deploy new contracts from an owned account
+
+account
+  Create new external accounts; This only has an effect when applied to global
```
Comment (chenette) [inline docs/source/seth_developers_guide/permissions.rst]: lower-case "this".  Sorry - I missed this in my first pass.

**Pull request merged**